docs/introduction-to-smart-contracts.rst

```diff
@@ -219,9 +219,9 @@ A blockchain is a globally shared, transactional database.
 This means that everyone can read entries in the database just by participating in the network.
 If you want to change something in the database, you have to create a so-called transaction
 which has to be accepted by all others.
-The word transaction implies that the change you want to make (assume you want to change
+The word transaction implies that the change you want to make (assume you want to e.g. change
```

Comment:
I would move the 'e.g.' to before the 'assume', or better say, 'for example, assume you want to…'.

Reply:
I think I'll just remove it again, it does not really fit.